cicd/docker/setup/config/dev/cube/schema/Events.js

```diff
@@ -1,213 +1,153 @@
 cube(`Events`, {
-  sql: `SELECT * FROM clickhouse_test_db.events`,
-
+  sql: `
+    WITH CountsAndLastURL AS (SELECT lookbackwindow, MAX(utc_time) AS maxDate
+                              FROM events
+                              GROUP BY lookbackwindow
+    )
+    SELECT
+      E.experiment,
+      E.lookbackwindow,
+      E.variant,
+      E.runningid,
+      toDateTime(toStartOfDay(toTimeZone(toDateTime(utc_time), 'UTC'), 'UTC'), 'UTC') as day,
+    min(CASE WHEN (isexperimentpage = true AND event_type = 'pageview') THEN utc_time END) as firstExperimentPageVisit,
+    max(CASE WHEN (istargetpage = true AND event_type = 'pageview') THEN utc_time END) as lastTargetPageVisit,
+    firstExperimentPageVisit is not null as isSession,
+    COUNT(CASE WHEN event_type = 'pageview' THEN 1 ELSE 0 END) AS pageviews,
+    SUM(CASE WHEN (E.isexperimentpage = true AND C.maxDate = E.utc_time AND event_type = 'pageview') THEN 1 ELSE 0 END) as experimentPageLastVisited
+    FROM events E JOIN CountsAndLastURL C ON C.lookbackwindow = E.lookbackwindow
+    WHERE event_type = 'pageview'
+    GROUP BY experiment, runningid, lookbackwindow, variant, day
+    having isSession = 1
+    order by day
+  `,
   preAggregations: {
     // Pre-Aggregations definitions go here
-    // Learn more here: https://cube.dev/docs/caching/pre-aggregations/getting-started  
+    // Learn more here: https://cube.dev/docs/caching/pre-aggregations/getting-started
```

Comment:
Will these be needed?

Reply:
remove https://github.com/dotCMS/core/pull/26586/commits/3b03fe7de9c28ae9bf5fbd50d743fa7917b39f10#diff-81c2b7651a61e07b6120a6b74f75a426dfda5d335f4d762b5d7bb83d4f6ba004L25